documentation/slate/source/includes/apis.md

```diff
@@ -1045,10 +1045,27 @@ You just have to verify that the coordinates of the geocoded object are inside t
 | nop       | from                    | id            | The id of the departure of your journey. Required to compute isochrones "departure after" |               |
 | nop       | to                      | id            | The id of the arrival of your journey. Required to compute isochrones "arrival before"    |               |
 | yep       | datetime                | [iso-date-time](#iso-date-time) | Date and time to go                                                                          |               |
+| nop       | max_duration            | int           | Maximum duration of isochrones in secondes                                                |               |
+| nop       | min_duration            | int           | Minimum duration of isochrones in secondes                                                |     0         |
+| nop       | <a name="traveler-type"></a>traveler_type | enum | Define speeds and accessibility values for different kind of people.<br>Each profile also automatically determines appropriate first and last section modes to the covered area. Note: this means that you might get car, bike, etc fallback routes even if you set `forbidden_uris[]`! You can overload all parameters (especially speeds, distances, first and last modes) by setting all of them specifically. We advise that you don't rely on the traveler_type's fallback modes (`first_section_mode[]` and `last_section_mode[]`) and set them yourself.<br><div data-collapse><p>enum values:</p><ul><li>standard</li><li>slow_walker</li><li>fast_walker</li><li>luggage</li></ul></div>| standard      |
```

Comment:
Agree with TeXitoi for the "see description in journeys".
And that way, you won't have to correct the traveler_type bug in documentation ;)